packages/mux-player/src/media-chrome/time-display.ts

```diff
@@ -99,6 +99,12 @@ class MxpTimeDisplay extends globalThis.HTMLElement {
       } else {
         this.timeDisplayEl?.removeAttribute('remaining');
       }
+    } else if (attrName === 'disabled' || (attrName === 'aria-disabled' && _oldValue !== newValue)) {
```

Comment:
suggestion(non-blocking): aside from the state change logic, this is effectively the same as the `remaining` condition. We should be able to simplify this to something like our `includes` check convention, i.e.:
```js
if (['remaining', 'disabled', 'aria-disabled'].includes(attrName) && oldValue !== newValue) {
// ...
```